include/deal.II/grid/tria.h

```diff
@@ -78,12 +79,10 @@ namespace internal
 {
   namespace TriangulationImplementation
   {
-    template <int dim>
     class TriaLevel;
```

Comment:
Since you `#include` the header file above, you don't need these forward declarations any more.

Reply:
I will don. I still have problems serializing/deserialzing `TriaLevel` (annoying Boost exception) so I did add it back here.